### ISSUE TITLE
calculate new loss

### ISSUE THREAD

**Issue by JensenLav** [JensenLav/Predicting-Privileged-Information#1]
modify loss function for sac. We need to add the new objective with its hyper-parameter

Loss = L_orig + lambda * Loss_new
